services/apis/requestor/requestorTasks.js

```diff
@@ -16,9 +16,26 @@ module.exports = {
     return reqID;
   },
 
+  async createRequestFromPolicyID(user, policyID, revoke = false) {
+    console.log(`### creating request for a policy id: ${policyID}`);
+    const endPoint = revoke ? `${studyViewerProps.endpoint.requestEndPoint}?revoke` : `${studyViewerProps.endpoint.requestEndPoint}`;
+    const getResponse = await I.sendPostRequest(
+      endPoint,
+      {
+        username: user.username,
+        policy_id: policyID,
+      },
+      user.accessTokenHeader,
```

Comment:
The user we are granting/revoking access for (username) is not necessarily the user who is creating the request (access token). To be clear about what is being tested and how the API works, it would be better to separate these 2. See in design doc; `we need a 2nd user that has access to update statuses in Requestor`. I would grant the access to create/update any request to a user A, and use other users B, C, etc that we are granting/revoking access to.